packages/xchain-thorchain/src/client.ts

```diff
@@ -613,6 +631,32 @@ class Client extends BaseXChainClient implements ThorchainClient, XChainClient {
       return getDefaultFees()
     }
   }
+
+  private async determineGasLmit({
```

Comment:
Pls rename `determineGasLmit` to `estimateGasLimit` or even better: Remove this function (see next comment below)

Reply:
done